contracts/src/v0.1/L2endpoint.sol

```diff
@@ -2,14 +2,22 @@
 pragma solidity ^0.8.16;
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "./interfaces/IL2Aggregator.sol";
+import "./VRFConsumerBase.sol";
 
 contract L2Endpoint is Ownable {
+    uint256 private constant GAS_FOR_CALL_EXACT_CHECK = 5_000;
     uint256 public aggregatorCount;
     uint256 public submitterCount;
-
+    uint64 sNonce;
+    struct RequestInfo {
+        address owner;
+        uint32 callbackGasLimit;
+    }
     mapping(address => bool) submitters;
     mapping(address => bool) aggregators;
 
+    mapping(uint256 => RequestInfo) internal sRequestDetail;
```

Comment:
We should keep the style of code same across the file and across the projects at best. Previously, you did not prefix storage variables with `s` prefix, but now `sRequestDetail` and `sNonce` do. How about making all to use the `s` prefix? It would then be compatible with the rest of off-chain codebase.